### DISCUSSION TITLE
8-layer MIMO support

### DISCUSSION THREAD

**Discussion by xfoukas** [srsran/srsRAN_Project#902] [General Help]
Hi,

I can see that 8-layer MIMO is on the roadmap. What is the timeframe for releasing this feature and will it support split 7.2?

Thanks!

**Comment by pgawlowicz** [comment]
Yes, it is on our roadmap. Please check [here](https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html)
